src/main/resources/assets/api-docs.yaml

```
@@ -365,6 +387,12 @@ definitions:
     example:
       { "generalUse": false, "hmbResearch": true, "diseaseRestrictions": ["http://purl.obolibrary.org/obo/DOID_602"] }
 
+  DataUseRecommendation:
+    type: object
+    description: JSON model of a set of data use recommendations
+    example:
+      { "http://purl.obolibrary.org/obo/DUO_0000042": { title: "General Research Use", "category", "Data Use Permission" } }
+
```

I think what you want here is this:
```suggestion
    example:
      { "http://purl.obolibrary.org/obo/DUO_0000042": {"title": "General Research Use", "category": "Data Use Permission" } }
```

This seems to correlate more closely with map of String -> Recommendation which is the response object.

---

We don't have a real `DataUseRecommendation` class. Perhaps we can create one and use that as the actual response object in the resource class?